test/svid/jwtsvid/test_jwt_svid.py

```diff
@@ -231,6 +231,29 @@ def test_parse_and_validate_invalid_missing_kid_header():
     assert str(exception.value) == 'key_id cannot be empty.'
 
 
+def test_parse_and_validate_invalid_missing_sub():
+    rsa_key = rsa.generate_private_key(public_exponent=65537, key_size=2048)
+    jwt_bundle = JwtBundle(TrustDomain('test.orgcom'), {'kid1': rsa_key.public_key()})
+    rsakeypem, _ = get_keys_pems(rsa_key)
```

Comment:
nit: `rsa_key_pem` looks clearer to me than `rsakeypem`.

An alternative to  `rsa_key_pem, _ = get_keys_pems(rsa_key)`is: `rsa_key_pem = get_keys_pems(rsa_key)[1]`

`get_keys_pems()` returns two values what means that is returning a tuple. That is why you can access what you need here using the index inside that returned tuple. I think this is it _pythonic_ way to do it.

Reply:
I understand this... but I have seem both in Python, so, I am not so sure this is the _pythonic_ way... I will make the variable change name.